inc/js_scripts/admin/main.js

```diff
@@ -91,6 +116,11 @@
 		render: function () {
 			this.setTitleColor( this.model.get( 'header_color' ) );
 			this.setContentColor( this.model.get( 'content_color' ) );
+			this.$el.find( '#site-logo' ).attr( 'src', post_modifier.settings.site_logo_src );
```

Comment:
I forgot to say this before, but you can write `this.$( selector )` as shorthand instead of `this.$el.find( selector )` for backbone views, they are the same thing